src/main/java/edu/wpi/first/wpilibj/networktables/NetworkTablesJNI.java

```diff
@@ -67,12 +55,6 @@ else if (System.getProperty("os.name").startsWith("Mac"))
     }
   }
 
-  public static final int NT_NET_MODE_NONE = 0x00;
-  public static final int NT_NET_MODE_SERVER = 0x01;
-  public static final int NT_NET_MODE_CLIENT = 0x02;
-  public static final int NT_NET_MODE_STARTING = 0x04;
-  public static final int NT_NET_MODE_FAILURE = 0x08;
-
```

Comment:
This seems like an unintentional revert.  Are you synced up to the latest master?